portal/views/clinician.py

```diff
@@ -15,15 +15,16 @@
 clinician_api = Blueprint('clinician_api', __name__)
 
 
-def clinician_query(acting_user, org_filter=None):
+def clinician_query(acting_user, org_filter=None, include_staff=False):
```

Comment:
Seems odd for this function to optionally query for users who have the staff role but not clinician or primary_investigator (the latter two are rather closely related, so that makes sense). Def not a blocker from my pov, I expect that refactoring something like this to be more general is prob not worth it.